vagrant-kubernetes-istio/RunTestOnHost/README.md

```diff
@@ -47,13 +47,25 @@ sh test_setup.sh
 You should push new images whenever you modify istio source code.
 
 ## 4. Run tests!
+### 1. Run tests on host machine
+You can issue test command on your host machine.
 E.g.
 ```bash
 cd $ISTIO/istio
 make e2e_simple E2E_ARGS="--use_local_cluster" HUB=10.10.0.2:5000 TAG=latest
 ```
 You can keep repeating this step if you made any local changes and want to run e2e tests again.
 Add E2E_ARGS="--use_local_cluster" to all your e2e tests as tests are we are running a local cluster.
+### 2. Run tests on VM
+You can also ssh to VM and issue test command within the VM.
+E.g.
+```bash
+cd $ISTIO/istio/vagrant-kubernetes-istio/RunTestOnHost
```

Comment:
I think we can remove this RunTestOnHost now right? put all files within vagrant-kubernetes-istio directory?

Reply:
The RunTestOnVM is what we want to remove. But it has introduction about using delve in VM. Which is missed in RunTestOnHost.